src/Tests/Microsoft.NET.TestFramework/TestAsset.cs

```diff
@@ -13,6 +13,10 @@
 
 namespace Microsoft.NET.TestFramework
 {
+    /// <summary>
+    /// A directory wrapper around the <see cref="TestProject"/> class.
+    /// It manages the on-disk files of the TestProject and provides additional functionality to edit projects.
+    /// </summary>
```

Comment:
This isn't only a wrapper over a `TestProject`.  It may also be a wrapper over files that are copied from the `src\Assets` folder.  It is created via `TestAssetsManager.CopyTestAsset` or `TestAssetsManager.CreateTestProject`, depending on which is the case.